src/components/dialog/index.tsx

```diff
@@ -1,41 +1,35 @@
 import { DialogOverlay, DialogContent } from '@reach/dialog'
-import { AnimatePresence, m } from 'framer-motion'
+import { AnimatePresence, m as motion, useReducedMotion } from 'framer-motion'
```

Comment:
Worth noting that `m` and `motion` are functionally different, this rename could cause confusion: https://www.framer.com/docs/guide-reduce-bundle-size/#how-to-reduce-bundle-size

Reply:
I'm finding just `m` to be confusing. Is there another name that's not `motion` that can help provide clarity?

Reply:
@BRKalow what do you think about `slimMotion`?

Reply:
Works for me 👍

Reply:
Boom, done: https://github.com/hashicorp/dev-portal/pull/626/commits/b7e53c815d14c5d6dcea2be879ba42a9880e3332 💥